Tasks/AzureRmWebAppDeployment/dependencies.json

```diff
@@ -10,6 +10,16 @@
               "MSDeploy3.6/Microsoft.Web.Deployment.Tracing.dll",
               "MSDeploy3.6/msdeploy.exe"
             ]
+        },
+		    {
+            "archiveName": "7zip.zip",
+            "url": "https://vstsagenttools.blob.core.windows.net/tools/7zip/1/7zip.zip",
```

Comment:
We need to add appropriate license and EULA files for this. Can you check with @RoopeshNair for this.